src/components/SideBar/SideBar.tsx

```diff
@@ -12,12 +14,14 @@ const linksForSideBar = [
   {title: 'Tags & Categories Distribution', pathname: '/tagged-and-category'},
 ];
 
+const startAtSelector = ({startAt}: AppState) => startAt;
+const endAtSelector = ({endAt}: AppState) => endAt;
+
 const SideBar: FC = () => {
+  const startAt = useSelector(startAtSelector);
```

Comment:
No need for two separate selectors you can handle state in one.

```suggestion
  const {startAt, endAt} = useSelector(({startAt, endAt}: AppState) => ({startAt, endAt})));
```